.trunk/configs/.bandit

```diff
@@ -0,0 +1,2 @@
+[bandit]
+exclude = tests
```

Comment:
`assert`s are used in tests and bandit says to exclude `tests` 🤷 
https://bandit.readthedocs.io/en/latest/config.html#exclusions